README.md

```diff
@@ -173,24 +195,11 @@ The table below indicates the models that are accessible at present along with a
 
 #### Contributors
 
-*Active maintainers*:
-- Anthony Blaom
-- Thibaut Lienart
-
-*Active collaborators*:
-- Diego Arenas
-- David Buchaca
-- Julio Hoffimann
-- Samuel Okon
-- Julian Samaroo
-- Sebastian Vollmer
-
-*Past collaborators*:
-- Ed Barp
-- Mosè Giordano
-- Franz Kiraly
-- Zac Nugent
-- Yiannis Simillides
+*Active maintainers*: A. Blaom, T. Lienart
+
+*Active collaborators*: D. Arenas, D. Buchaca, J. Hoffimann, S. Okon, J. Samaroo, S. Vollmer
+
+*Past collaborators*: D. Aluthge, E. Barp, G. Bohner, M. K. Borregaard, V. Churavy, H. Devereux, M. Giordano, M. Innes, F. Kiraly, M. Nook, Z. Nugent, P. Oleśkiewicz, A. Shridar, Y. Simillides, A. Sengupta, A. Stechemesser.
 
 #### License
```

Comment:
I have given this some thought, and I think it is important to acknowledge Sebastian and Franz in the design. Can we please add:

*Core design*: A. Blaom, F. Kiraly, S. Vollmer